packages/cli/commands/sandbox/sync.js

```diff
@@ -143,12 +145,24 @@ exports.handler = async options => {
   }
 
   try {
+    const availableSyncTasks = await getAvailableSyncTypes(
+      parentAccountConfig,
+      accountConfig
+    );
+    const syncTasks = await getSyncTypesWithContactRecordsPrompt(
+      accountConfig,
+      availableSyncTasks,
+      force
+    );
+
```

Comment:
Fetch and adjust sync tasks before passing into util

Reply:
OOC, what is the difference between `availableSyncTasks` and `syncTasks`? We only sometimes allow customers to sync contacts, is that correct? Why?

Reply:
Yes exactly. Long story short, we don't have a good way of syncing contacts multiple times due to ID changes, so we limit contact syncs to one for any sandbox. Our /types fetch will return a valid list of sync types, so if contacts have been synced already it won't be returned. Future plans are to change the sync behavior to support multiple syncs, but it may not be for a while